Index: app/src/main/java/com/example/quickcashapp/SubActivityJobPost.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.quickcashapp;\r\n\r\nimport android.os.Bundle;\r\n\r\nimport androidx.activity.EdgeToEdge;\r\n\r\npublic class SubActivityJobPost extends MainActivityEmployer {\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        EdgeToEdge.enable(this);\r\n        setContentView(R.layout.acitivty_sub_jobpost);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/quickcashapp/SubActivityJobPost.java b/app/src/main/java/com/example/quickcashapp/SubActivityJobPost.java
--- a/app/src/main/java/com/example/quickcashapp/SubActivityJobPost.java	(revision 00ac1d8749ba6a98167b1cc72e5fe45c6bbe0444)
+++ b/app/src/main/java/com/example/quickcashapp/SubActivityJobPost.java	(date 1728504089025)
@@ -11,4 +11,4 @@
         EdgeToEdge.enable(this);
         setContentView(R.layout.acitivty_sub_jobpost);
     }
-}
\ No newline at end of file
+}
